libs/ledgerjs/packages/react-native-hw-transport-ble/src/BleTransport.ts

```diff
@@ -10,8 +10,8 @@ import {
   getInfosForServiceUuid,
 } from "@ledgerhq/devices";
 import type { DeviceModel } from "@ledgerhq/devices";
-import { sendAPDU } from "@ledgerhq/devices/ble/sendAPDU";
-import { receiveAPDU } from "@ledgerhq/devices/ble/receiveAPDU";
+import { sendAPDU } from "@ledgerhq/devices/lib/ble/sendAPDU";
+import { receiveAPDU } from "@ledgerhq/devices/lib/ble/receiveAPDU";
```

Comment:
Why are we using `/lib` here ?

Reply:
This is the whole point of this PR 😉.

[`metro`](https://facebook.github.io/metro/) is unable to map `@ledgerhq/devices/lib/ble/sendAPDU` to either `/lib/…` or `/lib-es/…` because it does not support package exports.
Workarounds exists but it's not ideal to have to modify the metro config file.

Adding `/lib` here is not a big deal anyway since the `react-native-hw-transport-ble` package is only consumed by react-native clients and will never need to resolve the `/lib-es` version.